tests/unit/__init__.py

```diff
@@ -1332,14 +1332,19 @@ def setUp(self):
         self.content_request_questionnaire_people_in_household_title_en = 'How many people are in your household?'
         self.content_request_questionnaire_people_in_household_error_empty_en = \
             'Enter the number of people in your household'
-        self.content_request_questionnaire_people_in_household_error_nan_en = 'Enter a numeral'
+        self.content_request_questionnaire_people_in_household_error_nan_en = 'Enter a number'
+        self.content_request_questionnaire_people_in_household_error_number_less_en = \
+            'Enter a number less than 30'
```

Comment:
similarly here

Reply:
Changed flash message text to 31 throughout